### PR TITLE
Norvallen Region NM Audit

### PR DESCRIPTION
**Pull request by CriticalXI**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

- Lumber Jack will now instant cast spells and has double attack
- Tempest Tigon now has enwater/enaero
- Fradubio now spawns Duessa saplings when using Pinecone Bomb
- Fradubio and Fraelissa now have auto regen during the day
- King Arthro now instant casts spells
- Meteormauler Zhagtegg now has the chance to spawn with two guards that assist and follow him.
- Overlord Bakgodek now has TP drain on auto attacks
- Implemented the 7 Sins Skull NMs in The Eldieme Necropolis
- Several minor trait adjustments to other NMs
- Corrected HP, level and immunities for several NMs.
- Edited all drop lists to utilize the TH variables and correct drop rates according to FFXIDB.
- Edited all lottery NMs with a random respawn range to instead have a minimum respawn.

## Steps to test these changes

Test spawning of the skull NMs and any other NMs that have been adjusted.

### CODE REVIEW

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua:10]: Note: the guards don't always spawn with him: nobody knows why or what determines them spawning or not. The same is true for the other beastmen lair guardian NMs and their helpers

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: some things:

If cutting all damage types, there is a single mod that can be used for all of these. It doesn't have to be the uncapped version of each separately since mobs do not cap below 50% DT. BG specifies it as 50% magic damage, while clopedia doesn't.

File: scripts/zones/Jugner_Forest/mobs/Stag_Beetle.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Stag_Beetle.lua:18]: :cookie:

File: scripts/zones/Jugner_Forest/mobs/Sappy_Sycamore.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Sappy_Sycamore.lua]: EARTH_RES was recently renamed to EARTH_MEVA to reflect is actual effect and avoid problems when resistance rank mods get added later.

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (TeoTwawki) [inline scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua]: He doesn't have perma chainspell speed on, he casts very fast yes, but not instant. it is _very fast_ I'd estimate 75 to 90 eyeballing - if we could frame by frame a a few casts in a vid we can ballpark it from that vs a non-enhanced cast time.

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (TeoTwawki) [inline scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua:14]: Despite what clopedia says 
image
he probably gets this from a war subjob

if his nukes are garbage with a war sub, leave his sub alone and keep the mod - its conjecture either way, just looking for path of least resistance in mob building. I'll trust you judgement on this.

Comment (TeoTwawki) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua:14]: if his nukes are garbage with a war sub, leave his sub alone and keep the mod its conjecture either way, just looking for path of least resistance in mob building. I'll trust you judgement on this.

File: scripts/zones/Jugner_Forest/IDs.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/IDs.lua]: these are sequential, there isn't really a need to add the 2 helper mobs to the table. You can just `METEORMAULER +1` and `METEORMAULER +2` to refer to them. This is safe, they are unlikely to ever move/split up the group

File: scripts/zones/Jugner_Forest/IDs.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/IDs.lua]: similar to meteomaulers adds, you only really need to know the offset from Fradubio's ID

File: scripts/zones/Jugner_Forest/mobs/Fradubio.lua
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Fradubio.lua]: we definitely do not want magic numbers in script

Comment (TeoTwawki) [reply scripts/zones/Jugner_Forest/mobs/Fradubio.lua]: mob AI should automatically try and spread these out. if it doesn't I feel that's deserving of an issue.

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua
Comment (CriticalXI) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua:10]: Yeah if you see a bit below this line, I have a 50% chance for the guards to spawn with him. Since I didn't have a ton of info otherwise, I thought this would be the best implementation.

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (CriticalXI) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua:14]: His damage if unresisted still seems fine but taking off the BLM sub his magic accuracy is real bad. I think I would prefer to keep the BLM sub and just give him the double attack mod.

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua
Comment (CriticalXI) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: Got that adjusted to just one mod

File: scripts/zones/Jugner_Forest/mobs/Sappy_Sycamore.lua
Comment (CriticalXI) [reply scripts/zones/Jugner_Forest/mobs/Sappy_Sycamore.lua]: Corrected to EARTH_MEVA

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (CriticalXI) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua]: I reduced his Fastcast down to 75. Seems to match up what I see on videos of people fighting him.

Comment (CriticalXI) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua]: I have also added in a check to see whether or not Lumber Jack died or just despawned to set Weeping Willow's respawn correctly.

File: scripts/zones/Jugner_Forest/IDs.lua
Comment (CriticalXI) [reply scripts/zones/Jugner_Forest/IDs.lua]: Adjusted the IDs for both Meteormauler and Fradubio's saplings.

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (TeoTwawki) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua:14]: ok. thanks for checking.

File: scripts/zones/Jugner_Forest/mobs/Fradubio.lua
Comment (CriticalXI) [reply scripts/zones/Jugner_Forest/mobs/Fradubio.lua]: Took out that code, it worked fine without it.

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua
Comment (TeoTwawki) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua:10]: the comment is more here for educating passers by on your pr :+1: 

(I'd rejected attempts to make them always spawn)

File: scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua
Comment (TeoTwawki) [reply scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua]: > I have also added in a check to see whether or not Lumber Jack died or just despawned to set Weeping Willow's respawn correctly.

much appreciated, those of us that know this happens usually forget. Memory fuzzy, but I don't think its a static 30 min. But its acceptable till someone has concrete data.

**Review by TeoTwawki — APPROVED** [review]
everything that caught my eye has been adjusted, will want another set of eyes but lgtm

**Review by claywar — APPROVED** [review]

**Pull request merged**
